### PR TITLE
Deemphasize API Key ID in output message when creating key

### PR DESCRIPTION
**Pull request by joshma**
Users often use the key ID instead of the key value because the message
prints the key ID last.

This makes it easier to read the key instead:

```
  Creating API key named API Key (created 2021-09-14T16:11:27-07:00)...
  Done! Created API key key20210914zyyy.

  Save this somewhere safe, as you won't be able to retrieve it later:

  API Key: tkn_xxx

  (Team ID: xyz)
```

### CODE REVIEW

File: pkg/cmd/apikeys/create/create.go [hunk 1]
```diff
@@ -50,15 +50,15 @@ func run(ctx context.Context, c *cli.Config, name string) error {
 	}
 
 	apiKey := resp.APIKey
-	logger.Log("  Done!")
+	logger.Log("  Done! Created API key %s.", apiKey.ID)
```
Comment (colinking) [inline pkg/cmd/apikeys/create/create.go:53]: Does the API key ID provide any value to customers? If anything, should we debug log it instead?

File: pkg/cmd/apikeys/create/create.go [hunk 2]
```diff
@@ -50,15 +50,15 @@ func run(ctx context.Context, c *cli.Config, name string) error {
...
+		logger.Log(`
+  Save this somewhere safe, as you won't be able to retrieve it later:
 
-  %s
+  API Key: %s
 
-  Key ID: %s
-  Team ID: %s
-`, logger.Blue(apiKey.Key), apiKey.ID, apiKey.TeamID)
+  (Team ID: %s)
```
Comment (colinking) [inline pkg/cmd/apikeys/create/create.go:60]: The team ID is only useful when configuring an agent, right? What if we didn't print the team ID and instead updated the agent docs with a command for fetching the team ID? `airplane auth info`

We should also prioritize updating our api key format to include the team ID so folks don't even need that

**Review by colinking — APPROVED** [review]

File: pkg/cmd/apikeys/create/create.go (hunk 2 above)
Comment (joshma) [reply pkg/cmd/apikeys/create/create.go:60]: Hmmm yes, but we only really use api keys when setting up agents. But OK, I think `airplane auth info` will work. :)

File: pkg/cmd/apikeys/create/create.go (hunk 1 above)
Comment (joshma) [reply pkg/cmd/apikeys/create/create.go:53]: 👍

**Pull request merged**